src/main/java/sorald/Constants.java

```diff
@@ -46,12 +49,15 @@
     public static final String DOUBLE = "double";
 
     public static final String STRING_QUALIFIED_NAME = "java.lang.String";
+    public static final String JAVA_VERSION_SYSTEM_PROPERTY = "java.version";
 
     public static final String TOSTRING_METHOD_NAME = "toString";
     public static final String HASHCODE_METHOD_NAME = "hashCode";
 
     public static final String PATH_TO_STATS_OUTPUT = "experimentation/stats/warnings";
 
+    public static final String VIOLATION_USABLE_PATH_SEPARATOR = "###";
```

Comment:
This is not a good separator as hash symbols can be part of filepaths. It's also less convenient to split a path on multiple characters.

As we have ambitions to support both Windows and Unix-likes, we should use `File.pathSeparator` as the separator (`:` on most Unix-likes and `;` on Windows).

Reply:
Done!